### PR TITLE
Sberdoma 720 Displaying billing receipts data (lvl 1 only)

### PR DESCRIPTION
**Pull request by SavelevMatthew**
# Final state:
image
# No-billing state
image
# In progress state
image
# Error state
image
# No report state
image

### CODE REVIEW

File: apps/condo/domains/billing/utils/period.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+function getPeriodMessage (period, locale = 'default') {
+    if (!period || !locale) return
+    const date = new Date(period)
+    if (isNaN(date.getMonth())) return
+    const fullMonth = date.toLocaleString(locale, { month: 'long' })
+    return `${fullMonth} ${date.getFullYear()}`
+}
+
+function getPreviousPeriods (period, totalAmount = 3) {
```
Comment (Dimitreee) [inline apps/condo/domains/billing/utils/period.js:9]: can u use moment.js or date-fns for this periods generation?

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/pages/billing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    return (
+        <FeatureFlagRequired name={'billing'} fallback={<Error statusCode={404}/>}>
+            <Head>
+                <title>
+                    {BillingTitle}
+                </title>
+            </Head>
+            <PageWrapper>
+                <PageHeader title={<Typography.Title style={{ margin: 0 }}>{BillingTitle}</Typography.Title>}/>
+                <OrganizationRequired>
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/billing/index.tsx:47]: pls use `BillingPage.requiredAccess = OrganizationRequired` like others page

File: apps/condo/domains/billing/utils/period.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/billing/utils/period.js:14]: replaced with luxon 👍

File: apps/condo/pages/billing/index.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/pages/billing/index.tsx]: done 👍

File: apps/condo/package.json [hunk 3]
```diff
@@ -71,6 +73,7 @@
     "jspdf": "^2.3.1",
     "less": "^3.12.2",
     "lodash": "^4.17.21",
+    "luxon": "^2.0.2",
```
Comment (toplenboren) [inline apps/condo/package.json:76]: Right now we have 3 date libraries

Luxon, Moment and date-fns.

Comment (toplenboren) [reply apps/condo/package.json:76]: Why cant we simply use date-fns here and not add new depenency?

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/billing/utils/period.js (hunk 1 above)
Comment (toplenboren) [reply apps/condo/domains/billing/utils/period.js:14]: Why?

File: apps/condo/package.json (hunk 3 above)
Comment (SavelevMatthew) [reply apps/condo/package.json:76]: will remove moment and date-fns later. Use №1037 task from JIRA to see current status

**Pull request merged**